js/pull-requests.js

```diff
@@ -0,0 +1,68 @@
+//requirment 4
+// var api = "https://api.github.com/repos/rarmatei/js-exercises/pulls";
+
+// fetch(api).then(function(result) {
+//   result.json().then(function(repoResult) {
+//     repoResult.forEach(function(results) {
+//       var prList = document.getElementById("pull-requests-list");
+//       var newitem = document.createElement("li");
+//       var links = document.createElement("a");
+//       links.href = results.html_url;
+//       links.text = results.user.login;
+//       newitem.appendChild(links);
+//       prList.appendChild(newitem);
+//     });
+//   });
+// });
+
+// requirment 5
+var api = "https://api.github.com/repos/rarmatei/js-exercises/pulls";
+// fetch(api)
+//   .then(function(response) {
+//     return response.json();
+//   })
+//   .then(function(data) {
+//     data.forEach(function(x) {
```

Comment:
Side note on the commented out code: returning in a forEach function is not generally useful (it's not map).